testing/ncids-css-testing/stories/components/nci-hero/nci-hero.modifiers.stories.jsx

```diff
@@ -0,0 +1,7 @@
+export default {
+	title: 'components/Hero/Modifiers/Dark',
+	argTypes: {},
+};
+
+export * from "./dark.stories";
```

Comment:
Same note here, `export *  from..` breaks the upgrade - change these to the `export const NamedComponent...` from the other examples

change for all